scholarx/2022/index.html

```diff
@@ -401,14 +449,21 @@ <h1 class="display-3 text-center mb-5">FAQ for Mentees</h1>
                                             data-toggle="collapse"
                                             data-target="#collapseFive"
                                             aria-expanded="false"
-                                            aria-controls="collapseFive"
-                                    >
-                                        <div class="question-heading">
-                                            What are the eligibility requirements?
-                                        </div>
-                                        <div class="text-primary collapse-sign"><i
-                                                class="plusMinusSign fa fa-plus float-right"></i>
-                                        </div>
+                                            aria-controls="collapseFive">   
+                                        <table class="faq-table">
+                                            <tr>
+                                                <td>
+                                                    <div class="question-heading">
+                                                        What are the eligibility 
+                                                    </div>requirements?
```

Comment:
Here too @janithlahirukariyawasam

Reply:
**With "<br" tag** 
![image](https://user-images.githubusercontent.com/88109775/203829084-e43bbfb6-2fab-420b-ac35-9dcbe8dfaf36.png)
![in div br](https://user-images.githubusercontent.com/88109775/203829164-43d3ccd5-ad05-4ee4-9aec-3852aaf4dcee.png)

**Outside the div**
![image](https://user-images.githubusercontent.com/88109775/203829559-b8a450d8-ad8c-423c-832e-0bdf5774dae1.png)

![error not align out div](https://user-images.githubusercontent.com/88109775/203829328-601d0696-44a6-49d8-8301-57a88d5beeef.png)


**Issue is visible to the thin displays with that implementation also.**